.github/workflows/rust.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
-  GST_PLUGIN_FEATURE_RANK: pulsesink:NONE,alsasink:NONE,jacksink:NONE,fakesink:MAX
+  GST_PLUGIN_FEATURE_RANK: openalsink:NONE,openalsrc:NONE,pulsesink:NONE,alsasink:NONE,jackaudiosrc:NONE,jackaudiosink:NONE,fakesink:MAX
```

Comment:
Do you have `fakeaudiosink` ? Might be a better fit than `fakesink`

Reply:
Good question. How do I know what sinks are available?

Reply:
I don't see `fakeaudiosink` listed here for example: https://gstreamer.freedesktop.org/documentation/coreelements/fakesink.html?gi-language=c

Reply:
It's in -bad (staging area), https://gstreamer.freedesktop.org/documentation/debugutilsbad/fakeaudiosink.html?gi-language=c#fakeaudiosink-page

The best way to know if you have it is to check if its in the `gst-inspect-1.0` output.